docs/source/resources/graphql-glossary.md

```diff
@@ -0,0 +1,240 @@
+---
+title: GraphQL Glossary
+description: A comprehensive list of important GraphQL words and acronyms
+---
+
+
+We have put together a glossary of words and acronyms that you might hear or come across frequently in GraphQL land. Every developer and GraphQL enthusiast should be able to easily look up a particular term here when referenced in specs, reference guides, READMEs and within a codebase.
+
+
+<h2 id="security">Apollo</h2>
```

Comment:
Note - the IDs are wrong on these headings, so we should change those before publishing

Reply:
Oops! How did I forget? Thanks @stubailo